components/brave_ads/browser/ads_service_impl.h

```diff
@@ -17,6 +17,8 @@
 #include "bat/ads/ads_client.h"
 #include "brave/components/brave_ads/browser/ads_service.h"
 #include "brave/components/brave_ads/browser/background_helper.h"
+#include "brave/components/brave_rewards/browser/rewards_service.h"
```

Comment:
these headers should go in the .cc file - there is already a forward decl below for RewardsService